it/controllers/TaskListControllerISpec.scala

```diff
@@ -329,6 +329,7 @@ class TaskListControllerISpec extends ControllerISpec {
             .registrationApi.getSection[EligibilitySubmissionData](scheme.eligibilitySubmissionData)
             .registrationApi.getSection[ApplicantDetails](Some(validFullApplicantDetails))
             .registrationApi.getSection[TransactorDetails](Some(validTransactorDetails))
+            .registrationApi.getSection[BankAccount](Some(bankAccount))
```

Comment:
This is what seems to be breaking the test for state. this should move to the if condition block on line 334 (and not be stubbed here).